src/views/base_view.js

```diff
@@ -34,6 +37,11 @@ export default AmpersandView.extend({
     dom.hide(this.el);
   },
 
+  // Adjust the height of the iframe to match the height of the body.
+  adjustHeight () {
+    webChannel.sendAdjustHeight(document.body.offsetHeight);
```

Comment:
The body currently has 5px of padding, but there's no reason we can't move that to margin on `.index`.